Sources/StreamVideoSwiftUI/CallView/VisibilityThresholdModifier.swift

```diff
@@ -72,10 +72,10 @@ struct VisibilityThresholdModifier: ViewModifier {
 
         /// Check if the content view is vertically within the parent's bounds.
         let verticalVisible = (minY + requiredHeight <= bounds.maxY && minY >= bounds.minY) ||
-                              (maxY - requiredHeight >= bounds.minY && maxY <= bounds.maxY)
+        (maxY - requiredHeight >= bounds.minY && maxY <= bounds.maxY)
```

Comment:
Indentation here and on the next one

Reply:
I actually did that in purpose to help with the readability of the calculations. I can roll it back if you want though.